src/index.ts

```diff
@@ -33,10 +33,11 @@ export function webhookHandler(req: Request, res: Response) {
 function handleWebhookRequest(request: Request, response: Response) {
     let agent = new WebhookClient({ request: request, response: response })
     LOG.debug({ intent: agent.intent, contexts: agent.contexts }, 'debug message')
-
+    
     let intentMap = new Map()
     intentMap.set('order-delivery-tracking_order-email', new OrderTrackingIntentHandler(agent, LOG).handleOrderTracking())
     intentMap.set('product-lookup_product-details', new ProductLookupIntentHandler(agent, LOG).handleProductLookup())
-
+    intentMap.set('Order product', new ProductLookupIntentHandler(agent, LOG).handleProductLookup())
```

Comment:
prev message more suitable here, please create dedicated intent handler instead of modification of existing one, if something is needed for reuse just extract into dedicated service class